src/js/components/LogRow.js

```diff
@@ -19,51 +20,69 @@ type Props = {
   log: Log,
   columns: TableColumns,
   onClick: () => void,
+  onDoubleClick: () => void,
   rightClick: RightClickBuilder
 }
 
-export default class LogRow extends React.Component<Props> {
-  shouldComponentUpdate(nextProps: Props) {
-    return (
-      !Log.isSame(this.props.log, nextProps.log) ||
-      !isEqual(this.props.columns, nextProps.columns) ||
-      this.props.highlight !== nextProps.highlight ||
-      this.props.dimens.rowWidth !== nextProps.dimens.rowWidth ||
-      this.props.timeZone !== nextProps.timeZone ||
-      this.props.timeFormat !== nextProps.timeFormat
-    )
-  }
+const LogRow = (props: Props) => {
+  const {
+    dimens,
+    highlight,
+    index,
+    log,
+    rightClick,
+    columns,
+    onClick,
+    onDoubleClick
+  } = props
 
-  render() {
-    const {dimens, highlight, index, log, rightClick, columns} = this.props
-    const renderCell = (column, colIndex) => {
-      const width = dimens.rowWidth !== "auto" ? column.width || 300 : "auto"
-      const field = log.field(column.name)
-      const key = `${index}-${colIndex}`
-      if (field) {
-        return (
-          <LogCell
-            rightClick={rightClick}
-            key={key}
-            field={field}
-            log={log}
-            style={{width}}
-          />
-        )
-      }
-      if (dimens.rowWidth !== "auto") {
-        return <div className="log-cell" key={key} style={{width}} />
-      }
-    }
+  const handleClick = useDoubleClick(onClick, onDoubleClick)
 
-    return (
-      <div
-        className={classNames("log-row", {highlight, even: index % 2 == 0})}
-        style={Styler.row(dimens)}
-        onClick={this.props.onClick}
-      >
-        {columns.getVisible().map(renderCell)}
-      </div>
-    )
+  const renderCell = (column, colIndex) => {
+    const width = dimens.rowWidth !== "auto" ? column.width || 300 : "auto"
+    const field = log.field(column.name)
+    const key = `${index}-${colIndex}`
+    if (field) {
+      return (
+        <LogCell
+          rightClick={rightClick}
+          key={key}
+          field={field}
+          log={log}
+          style={{width}}
+        />
+      )
+    }
+    if (dimens.rowWidth !== "auto") {
+      return (
+        <div
+          className="log-cell"
+          key={key}
+          style={{width}}
+          onClick={handleClick}
```

Comment:
I don't see why the log cell needs to have the click handler? Is that intentional?

Reply:
Ah shoot, no this is a mistake. I put it here at first and realized it belonged on the whole row but forgot to remove this. Nice catch